src/components/common/ActionModal.vue

```diff
@@ -246,6 +254,11 @@ export default {
     balance() {
       return this.liquidAtoms
     },
+    totalInvoice() {
+      return (
+        Number(this.amount) + Number(this.gasPrice) * Number(this.gasEstimate)
```

Comment:
@fedekunze @faboweb Is calculation correct?

Reply:
Looks good!